synapse/handlers/oidc.py

```diff
@@ -935,17 +952,32 @@ async def handle_redirect_request(
 
         state = generate_token()
         nonce = generate_token()
+        code_verifier = ""
 
         if not client_redirect_url:
             client_redirect_url = b""
 
+        metadata = await self.load_metadata()
+
+        # Automatically enable PKCE if it is supported. (Note we verified it contains S256 earlier.)
+        extra_grant_values = {}
+        if metadata.get("code_challenge_methods_supported"):
+            code_verifier = generate_token(48)
+
+            # Default to the values for plain.
```

Comment:
This doesn't sound correct?

```suggestion
            # Default to the values for S256.
```

Reply:
This just seems to be out of date, will reword!

Reply:
Done in 1e82afd7e88d9194b368519ecd696adbca4c856f.